app/assets/stylesheets/entities_form.css

```diff
@@ -0,0 +1,45 @@
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 30px;
+}
+
+.field,
+.field-two {
+  padding-bottom: 20px;
+}
+
+.field label {
+  padding-right: 28px;
+  font-size: 20px;
+}
+
+.field-two label {
+  padding-right: 10px;
+  font-size: 20px;
+}
+
+.field-three {
+  margin-top: 20px;
+  width: 100vw;
+  border: solid 1px #60b625;
+  height: 60px;
```

Comment:
- [ ] [OPTIONAL]
Nice job, app is running correctly 👏 However, let's take note that it's one of the project's objectives that our design closely mimics the given [template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=). Let's therefore ensure that the _typography_, _layout_ and _color_ on the splash screen is similar to the original design.

**Your splash screen:**     |  **Expected**     
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/shm04/budget-app/assets/12745474/f2e5d905-dc8e-461e-abdc-190787b40687"> | <img width="800" alt="Home Page" src="https://github.com/shm04/budget-app/assets/12745474/eeb6ab95-0fb7-48e2-8e40-d8f296b973b8">